addons/knowledge/static/src/webclient/commands/command_category.js

```diff
@@ -0,0 +1,8 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { _lt } from "@web/core/l10n/translation";
+
+const commandCategoryRegistry = registry.category("command_categories");
+commandCategoryRegistry.add("knowledge_articles", { namespace: "?" }, { sequence: 10 });
+commandCategoryRegistry.add("knowledge_extra", { namespace: "?" }, { sequence: 20 });
```

Comment:
what is knowledge_articles and knowledge_extra ?
We could add some comments here to explain what is what.
If I understand well:
knowledge_articles -> look for knowledge article and list them when searching using "?" prefix
knowledge_extra -> Add an advanced search button to the result list  when searching using "?" prefix